src/Uno.Toolkit.UI/Controls/NavigationBar/NavigationBar.xaml

```diff
@@ -40,10 +41,10 @@
 
 	<Style x:Key="DefaultMainCommandStyle"
 		   TargetType="AppBarButton">
-		<not_mobile:Setter Property="Icon">
-			<not_mobile:Setter.Value>
-				<BitmapIcon UriSource="{StaticResource BackButtonUriSource}" />
-			</not_mobile:Setter.Value>
+		<not_mobile:Setter Property="Icon" win:Value="{x:Null}">
```

Comment:
i would add a comment here saying "the default native back icon is used"
same for the v2 counterpart